hoomd/conftest.py

```diff
@@ -71,6 +71,7 @@ def make_simulation(snapshot=None, domain_decomposition=None):
                 sim.create_state_from_snapshot(snapshot)
             else:
                 sim.create_state_from_snapshot(snapshot, domain_decomposition)
+        sim.seed = numpy.random.randint(100_000)
```

Comment:
I see, this is the change is why the see test no longer users `simulation_factory`. Use a fixed seed so that failures are as repeatable as possible.